README.md

```diff
@@ -1,94 +1,20 @@
-# Kubemote
-
-A compact Kubernetes API library
+# Extended K8s deployments info CLI
```

Comment:
The previous `readme.md` was a better fit for the `kubemote` client library. Please update the readme file for the new project (deployments info) following the project split.